Codigo/InstanceManager.bas

```diff
@@ -0,0 +1,81 @@
+Attribute VB_Name = "InstanceManager"
+Option Explicit
+
+
+Private AvaibleInstanceMap As t_IndexHeap
```

Comment:
typo should be renamed to `AvailableInstanceMap`

Reply:
done